solc/CommandLineInterface.cpp

```diff
@@ -387,14 +387,12 @@ bool CommandLineInterface::readInputFiles()
 	{
 		if (!boost::filesystem::exists(m_fileReader.basePath()))
 		{
-			serr() << "Base path does not exist: " << m_fileReader.basePath() << endl;
-			return false;
+			BOOST_THROW_EXCEPTION(CommandLineValidationError() << errinfo_comment("Base path does not exist: "));
```

Comment:
The new message no longer contains `m_fileReader.basePath()`.

Same below and some other places. Please make sure that the exceptions still contain all the information that used to be printed to stderr.